Source/UI/NewUI/BillingForm/View/BillingFormPhoneNumberText.swift

```diff
@@ -52,21 +61,22 @@ final class BillingFormPhoneNumberText: BillingFormTextField {
     }
     
     // TODO: Copied from old code. Needs to be refactored
-    private func updateTextFieldFormat(textField: UITextField) {
+    /// Called when the Billing form Phone number text field is changed.
+    @objc func updateTextFieldFormat(textField: UITextField) {
```

Comment:
no longer private? Can't find where its being set, not it also has an `objc` tag?

Reply:
please suggest code using <cmd+g>